src/main/kotlin/no/nav/k9/los/nyoppgavestyring/mottak/oppgave/AktivOppgaveRepository.kt

```diff
@@ -236,13 +236,13 @@ class AktivOppgaveRepository(val oppgavetypeRepository: OppgavetypeRepository) {
             )
         }
 
-        private fun oppdaterFelter(updates: Map<Long, Verdi>, tx: TransactionalSession) {
+        private fun oppdaterFelter(updates: Map<Long, Verdi>, oppgave: OppgaveV3, tx: TransactionalSession) {
             if (updates.isEmpty()) {
                 return
             }
             tx.batchPreparedNamedStatement(
-                "update oppgavefelt_verdi_aktiv set verdi = :verdi, verdi_bigint = :verdi_bigint where id = :id",
-                updates.map { mapOf("id" to it.key, "verdi" to it.value.verdi, "verdi_bigint" to it.value.verdiBigInt) }.toList()
+                "update oppgavefelt_verdi_aktiv set verdi = :verdi, verdi_bigint = :verdi_bigint, oppgavestatus = cast(:status as oppgavestatus) where id = :id",
```

Comment:
Har ikke så for seg denne endringen, siden feltet ikke brukes og aktiv-tabellene er på vei ut.